.gitignore

```
@@ -1 +1,5 @@
+<<<<<<< HEAD
 node_modules/
+=======
+node_modules/
+>>>>>>> c7ef6403f5dbe9d8a76d66fd7e316ee59e8d0a9b
```

You have remainings of merge conflict here. It should only be:

```suggestion
node_modules/
```